packages/aws-client/src/StepFunctions.ts

```diff
@@ -28,15 +42,14 @@ export const doesExecutionExist = (describeExecutionPromise: Promise<unknown>) =
  * If a ThrottlingException is received, this function will retry using an
  * exponential backoff.
  *
- * @param {Object} params
- * @returns {Promise<Object>}
+ * @param {DescribeExecutionInput} params
+ * @returns {Promise<DescribeExecutionOutput>} DescribeExecutionOutput
  *
  * @kind function
  */
-export const describeExecution = improveStackTrace(
-  retryOnThrottlingException(
-    (params: AWS.StepFunctions.DescribeExecutionInput) => sfn().describeExecution(params).promise()
-  )
+export const describeExecution = retryOnThrottlingException(
```

Comment:
We're removing the `improveStackTrace` call here. What's the reasoning? Is it no longer necessary with the v3 functions?

Reply:
Per Charles and Jenny, v3 has better stack track built in, so they have been removing "improveStackTrace" in their v3 updates.

Reply:
Neat! We can probably remove that method once everything is switched over 🤔